bluesky_queueserver/manager/manager.py

```diff
@@ -1521,8 +1522,7 @@ def run(self):
         Overrides the `run()` function of the `multiprocessing.Process` class. Called
         by the `start` method.
         """
-
-        logging.basicConfig(level=logging.WARNING)
+        logging.basicConfig(level=max(logging.WARNING, self._log_level))
```

Comment:
What is the intention here? E.g. if `self._log_level` is `INFO` (=20), the set level will be `WARNING` (=30). The description says the default is `INFO`.

Reply:
This would be logging level for other packages except `bluesky_queueserver` for which logging is not set explicitly. Typically it is undesirable to get debug/info logging from all packages in any circumstances. So here I limit logging level to WARNING or higher.